apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -420,7 +420,7 @@ private void processAsync(
         record.setLastUpdated(Optional.of(loadedBatchBuilder.getTimestamp()));
 
         // Associate the beneficiary with this file loaded
-        loadedBatchBuilder.associateBeneficiary(rifRecordEvent.getBeneficiaryId());
+        loadedBatchBuilder.associateBeneficiary(String.valueOf(rifRecordEvent.getBeneficiaryId()));
```

Comment:
Should associateBeneficiary just be changed to long too?

Reply:
Great question. I've pushed it one level deeper now but the issue in going further is that we collect a bunch of beneficiaries and build a string out of it to store as a list in the database so the underlying entity will probably not be changing (and definitely not changing as part of the current scope of work) so I'm going to keep it as string in the LoadedBatch* classes to be internally consistent within that layer.